charts/u4a-component/charts/addon-component/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.1
+version: 0.1.2
```

Comment:
https://github.com/kubebb/components/blob/1c6e60d29f89eb86a113235cf75345e5d4f088af/charts/u4a-component/Chart.yaml#L28

This needs to be updated as well.